features/elasticsearch/event-forwarder/src/main/java/org/opennms/features/elasticsearch/eventforwarder/ForwardingEventListener.java

```diff
@@ -99,6 +100,15 @@ private void installMessageSelectors() {
 		getEventIpcManager().addEventListener(this);
 	}
 
+	public void destroy() {
+		Assert.notNull(eventIpcManager, "eventIpcManager must not be null");
```

Comment:
Asserting in the destroy method is unusual, normally we just skip the destroy/removal if the fields are found to be null.